types/node-unifi/index.d.ts

```diff
@@ -10,9 +10,91 @@ declare namespace nodeUnifi {
 
     interface SiteStats {
         _id: string;
+        external_id: string;
+        attr_no_delete: boolean;
+        attr_hidden_id: string;
         name: string;
         desc: string;
-        [key: string]: any;
+        num_new_alarms: number;
+        health: (
+            | {
+                subsystem: "wlan";
+                num_user: number;
+                num_guest: number;
+                num_iot: number;
+                "tx_bytes-r": number;
+                "rx_bytes-r": number;
+                status: string;
+                num_ap: number;
+                num_adopted: string;
+                num_disabled: number;
+                num_disconnected: number;
+                num_pending: number;
+            }
```

Comment:
1. Suggest extract each of the "subsystem" into its own interface.

```ts
interface WlanSubsystemHealrgStats {
    ...
}

...

health: WlanSubsystemHealthStats | ...;
```

2. Would be nice if you can help add test case for `SiteStats` and the new interfaces you created.

```ts
declare let stats: SiteStats;

// $ExpectType string
stats.external_id;
// ... and more test

// $ExpectType WlanSubsystemHealthStats | ...
stats.health;

declare let wlanSubsystemHealthStats: WlanSubsystemHealthStats;
// $ExpectType ...
wlanSubsystemHealthStats.subsystem;
// and test all other attributes in your added interfaces.
```